CommonKit/Sources/CommonKit/Helpers/AdamantSecureStorage.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  AdamantSecureStorage.swift
+//
+//
+//  Created by Stanislav Jelezoglo on 02.08.2024.
+//
+
+import Foundation
+
+final class AdamantSecureStorage: SecureStorageProtocol {
```

Comment:
Let's use struct: https://developer.apple.com/documentation/swift/choosing-between-structures-and-classes